fw/cache.c

```diff
@@ -308,7 +309,7 @@ __cache_method_test(tfw_http_meth_t method)
 	return cache_cfg.methods & (1 << method);
 }
 
-bool
+static inline bool
```

Comment:
Got it, fixed in the all files. Also removing it works like detector for unused functions :)

Reply:
It might be me but I think the removal of `inline` everywhere may have been overzealous.

Reply:
What do you mean by overzealous, what exactly drawbacks are?

Reply:
I meant that you had changed **all** inline functions in the code to regular. They have been inlined for a purpose, and I am not convinced the compiler will substitute all calls to former inline functions with their actual code (i.e. inline them) in all cases. It might do that for smaller functions called once or twice, and might not do that for functions called more frequently or for larger functions where the choice for code duplication (inlining) was made intentionally. All in all, I am not totally convinced by this global change. @krizhanovsky would know better on this subject. :-)

Reply:
I meant that it just didn't make sense to add `inline` specifier in this particular pull request. There is no sense to remove all inline specifiers around the project.

Reply:
It wasn't around the project but only in *.c files, but anyway, restored back.